Sources/Tools/plPythonPack/PythonInterface.cpp

```diff
@@ -121,7 +121,25 @@ void PythonInterface::finiPython()
 //
 PyObject* PythonInterface::CompileString(const char *command, const plFileName& filename)
 {
-    PyObject* pycode = Py_CompileString(command, filename.AsString().c_str(), Py_file_input);
+    PyObject* filenameObj = PyUnicode_FromStringAndSize(
+        filename.AsString().c_str(),
+        filename.AsString().size()
+    );
+
+    PyCompilerFlags flags{};
+    flags.cf_feature_version = PY_MINOR_VERSION;
+
+    // Py_CompileString decodes the filename using the active codepage, but
+    // we always use UTF-8 internally, so pass the filename as a Python string object.
+    PyObject* pycode = Py_CompileStringObject(
+        command,
+        filenameObj,
+        Py_file_input,
+        &flags,
+        2
```

Comment:
This explicitly sets the optimization level to `2` (i. e. remove Python `assert`s and docstrings) whereas previously it wasn't explicitly set. Is this safe? I *hope* we don't have any load-bearing `assert`s and docstrings, but we might need to test that... (The stub generator obviously needs docstrings, but that's not important when running with packed Python.)

According to [the docs](https://docs.python.org/3/c-api/veryhigh.html#c.Py_CompileString), `-1` would match the previous behavior.

Reply:
I was hoping to remove the docstrings indeed - we don't need those in python.pak. Assertions should probably be using `PtAssert()`.